src/ToDoList.js

```diff
@@ -135,20 +133,25 @@ const ToDoList = () => {
             >
               <ListItemIcon>
                 <Checkbox
-                  checked={task.completed}
+                  checked={task.isCompleted}
                   onChange={() => {
                     const updatedTasks = newTaskValue.map((t) =>
-                      t.id === task.id ? { ...t, completed: !t.completed } : t
+                      t.id === task.id
+                        ? { ...t, isCompleted: !t.isCompleted }
+                        : t
                     );
                     setNewTaskValue(updatedTasks);
-                    localStorage.setItem("", JSON.stringify(updatedTasks));
+                    localStorage.setItem(
+                      "newTaskValue",
+                      JSON.stringify(updatedTasks)
+                    );
                   }}
                 />
               </ListItemIcon>
               <ListItemText
                 primary={task.task}
                 primaryTypographyProps={{
-                  className: task.completed ? completedTaskStyle : "",
+                  className: task.isCompleted ? completedTaskStyle : "",
```

Comment:
you can use the `cx` function provided by `@emotion/css` in order to set multiple classnames based on conditions.
for this specifc case you can do `cx(task.isCompleted && completedTaskStyle);`

Reply:
Cool.